### PR TITLE
Migrate with ActiveRecord

### PR DESCRIPTION
**Pull request by alexander-myltsev**

### CODE REVIEW

File: db-migration/db/migrate/20160608180550_create_name_strings.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+class CreateNameStrings < ActiveRecord::Migration
+  def change
+    create_table :name_strings, id: false do |t|
+      t.uuid :id, null: false, primary_key: true
+      t.uuid :id_mysql, null: false
+      t.string :name, null: false, limit: 255
+      t.uuid :canonical_uuid
+      t.string :canonical, limit: 255
+      t.timestamps
```
Comment (dimus) [inline db-migration/db/migrate/20160608180550_create_name_strings.rb]: may be we should get rid timestamps

File: db-migration/db/migrate/20160609164952_create_name_string_indices.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class CreateNameStringIndices < ActiveRecord::Migration
+  def change
+    create_table :name_string_indices, id: false do |t|
+      t.integer :data_source_id, null: false
+      t.uuid :name_string_id, null: false
+      t.string :url, limit: 255
+    end
+    add_index :name_string_indices, :data_source_id, using: 'btree'
```
Comment (dimus) [inline db-migration/db/migrate/20160609164952_create_name_string_indices.rb]: is btree the fastest?

Comment (alexander-myltsev) [inline db-migration/db/migrate/20160609164952_create_name_string_indices.rb]: For a while. I would optimise such things a bit later. What is more appropriate index that you suggest?

Comment (dimus) [inline db-migration/db/migrate/20160609164952_create_name_string_indices.rb]: in MySQL world Percona makes some new developments with Fractal Trees https://en.wikipedia.org/wiki/TokuDB

Comment (alexander-myltsev) [inline db-migration/db/migrate/20160609164952_create_name_string_indices.rb]: Interesting. But not supported by Postrgres due to patents. http://postgresql.nabble.com/Fractal-tree-indexing-td5744987.html

**Pull request merged**